gitfiti.py

```diff
@@ -207,7 +207,7 @@ def load_images(img_names):
         return loaded_imgs
 
 
-def get_calendar(username, base_url):
+def retrieve_contributions_calendar(username, base_url):
     """retrieves the GitHub commit calendar data for a username"""
     base_url = base_url + 'users/' + username
```

Comment:
this seems wrong to me now, does base_url have a trailing slash? and will it always even if user entered?